symless/synergy-odin
# Update tooltips with more info

Upstream: https://github.com/deskflow/deskflow/pull/7788

https://symless.atlassian.net/browse/S1-1860

To test:
1. Click Edit
2. Click Preferences
3. Click Advanced
4. Hover over service checkbox
5. Expect: Tooltip
6. Hover over service combobox
5. Expect: Tooltip
4. Right click service checkbox
5. Expect: What's This (click should show more info)
5. Right click service combobox
5. Expect: What's This (click should show more info)
